spec/controllers/submissions_controller_spec.rb

```diff
@@ -20,18 +22,21 @@
       it { should redirect_to(proposal_path(assigns(:submission).proposal)) }
     end
 
-    context 'with invalid attributes' do
-      let(:invalid_submission) { double }
+    context 'with a valid submission attribute and invalid event and proposal id attributes' do
```

Comment:
Hi @flyinggrizzly — a bit confused about the wording here.

What does the `valid submission attribute` refer to? And isn't the proposal id valid here?

Reply:
Shoot. Yea, got myself mixed up. That should be `context 'with valid proposal attribute and invalid event and result attributes'`

I'll fix this and push again.